### DISCUSSION TITLE
Increase paid action invoice expirations when amount >= 1m sats?

### DISCUSSION THREAD

**Discussion by huumn** [stackernews/stacker.news#3026] [Ideas]
Or maybe just do it for pessimistic invoices?

https://stacker.news/items/689767/r/k00b?commentId=689999

**Comment by ekzyis** [comment]
Another option: always do it when it's not a wrapped invoice but they pay us directly like territory payments

That would overlap with some pessimistic (anon stuff) and optimistic invoices (posts and replies).

But if amount >=1m sats also sounds good
